src/main.rs

```diff
@@ -340,8 +341,12 @@ fn create_main(xi_peer: XiPeer) -> Result<(HWND, Rc<Box<WndProc>>), Error> {
         let main_state = MainWinState::new();
         let main_win: Rc<Box<WndProc>> = Rc::new(Box::new(
             MainWin::new(xi_peer, main_state)));
-        let width = 500;  // TODO: scale by dpi
-        let height = 400;
+
+        // Simple scaling based on System Dpi (96 is equivalent to 100%)
+        let dpi = GetDpiForSystem() as i32;
+        let width = 500 * (dpi/96);
```

Comment:
Yeah. I realized the same after posting the PR. I will do a work around here.
As for the per monitor thing, I agree with your experimentation :)